pyre/bench/synth/c_call_reports_the_callee_frame_not_the_inliners.py

```
@@ -0,0 +1,131 @@
+
+def outer_arming(n, hook):
+    t = 0
+    for _ in range(n):
+        t = inner_arming(t, hook) % 1000003
+    return t
+
+
+def collect():
+    seen = {}
+
+    def hook(frame, event, arg):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `hook`

Add return type annotation: `None`

(ANN202)